corehq/tests/nose.py

```diff
@@ -297,5 +301,23 @@ def find_module(self, name, path=None):
     # Register the import hook. See https://www.python.org/dev/peps/pep-0302/
     sys.meta_path.append(InfoImporter())
 
+
+@unit_testing_only
+def empty_databases():
+    """
+    Best effort at emptying all documents from all databases.
+    Useful when you break a test and it doesn't clean up properly. This took
+    about 5 seconds to run when trying it out.
+    """
+    for db in get_preindex_plugin('couchapps').get_dbs('all_docs'):
```

Comment:
:+1: I just copied this from somewhere, but that does look like a better approach.  Will update.